lib/openapi_first/test/coverage.rb

```diff
@@ -12,33 +16,64 @@ module Coverage
 
       Result = Data.define(:plans, :coverage)
 
-      @current_run = {}
-
       class << self
-        attr_reader :current_run
-
         def install = Test.install
 
         def start(skip_response: nil, skip_route: nil)
-          @current_run = Test.definitions.values.to_h do |oad|
-            plan = Plan.for(oad, skip_response:, skip_route:)
-            [oad.key, plan]
-          end
+          tracker = Tracker.new(Test.definitions, skip_response:, skip_route:)
```

Comment:
I am looking for code to distinguish the main process from "remote" process… but that is not needed here, because `Test.start` only run in the main process?

Reply:
> If you want to we can add something like Test.in_main_process? and guard […] What do you think? Alternatively we could also return early from Coverage.start if @drb_uri is non-nil?

Hey @richardboehme, I think both approaches are fine. I don't have a strong opinion on this things right now. Maybe the second variant is simpler and respects the lines between Test and Coverage? Do you want to add that guard in this PR?

> I thought about this as well while exchanging the current_run Hash with the Tracker class. 

I will remove `Coverage.current_run, .plans, .install, .uninstall` in another PR.

Reply:
Okay I added a small guard on top of the `Coverage.start` method!